x-pack/metricbeat/metricbeat.reference.yml

```diff
@@ -707,7 +707,7 @@ metricbeat.modules:
 #------------------------------ ZooKeeper Module ------------------------------
```

Comment:
It seems that this file in `x-pack` must be manually updated now. @ruflin do you think that I should open an issue to automate this when running `make update` from OSS?

Reply:
manually = running `mage update` in the x-pack directory or manually you mean by hand and CI will not detect it has missing content?

What behaviour do we have in filebeat here?

Reply:
Sorry, I mean by _manually_ running `mage update` in `x-pack` folder. Opening issue https://github.com/elastic/beats/issues/10369